.github/workflows/node.js.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm ci
+    - run: npm run build --if-present
+    - run: npm test
```

Comment:
I don't see any of those scripts [here](https://github.com/metaplex/arweave-cost/blob/main/package.json#L10-L14) except `test`

Also looks like we'd need to run `install` first?
Are we using `npm` across the board?
Otherwise we should consider using `yarn` which from my experience installs much faster.

Reply:
> npm ci will install

Ah nice .. didn't know about that one :)

Reply:
this is the exact github template for node. i didn't change anything we can tweak however but wanted to land

`npm ci` installs too

re: npm or yarn. we haven't discussed a preference

Reply:
I understand WRT CI and that we're just trying to get something to get going, however I'd expect this to fail ATM since some of the other scripts executed in the template don't exist in the `package.json`.
I guess I'm not clear of the goal here is it to get a _working_ action in or just something to get started with?

Reply:
`npm ci` is built into npm. 

`npm run buid --if-present` only executes if the script is declared

`npm test` is built in and will run the defined test script

happy to convert to yarn

Reply:
npm -> yarn: https://github.com/metaplex/arweave-cost/pull/2